tests/test_addons.py

```diff
@@ -442,6 +477,178 @@ def test_add_file_osfstorage_log(self):
         assert_equal(self.node.logs.count(), nlogs + 1)
         assert('urls' in self.node.logs.filter(action='osf_storage_file_added')[0].params)
 
+    def test_add_log_updates_cache_create(self):
+        self.configure_osf_addon()
+        path = 'lollipop'
+        url = self.node.api_url_for('create_waterbutler_log')
+        payload = self.build_payload(metadata={
+            'materialized': path,
+            'kind': 'file',
+            'path': path,
+            'size': 100,
+            'nid': self.node._id,
+        })
+        self.app.put_json(url, payload, headers={'Content-Type': 'application/json'})
+        self.node.reload()
+        assert self.node.storage_usage == 100
+
+    def test_add_log_updates_cache_update(self):
+        self.configure_osf_addon()
+        path = 'lollipop'
+        url = self.node.api_url_for('create_waterbutler_log')
+        payload = self.build_payload(metadata={
+            'materialized': path,
+            'kind': 'file',
+            'path': path,
+            'size': 120,
+            'nid': self.node._id,
+        }, action='update')
+        self.app.put_json(url, payload, headers={'Content-Type': 'application/json'})
+        self.node.reload()
+        assert self.node.storage_usage == 120
+
+    def test_add_log_updates_cache_move(self):
```

Comment:
Can you add a test for rename? Also, would it be reasonable to test moving from non-OSF to OSF and vice versa?

Reply:
👌